### PR TITLE
Add missing semicolons and ending metatag

### PR DESCRIPTION
**Pull request by antonybudianto**

### CODE REVIEW

File: routing.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ You may exclude URIs by adding them to the `$except` property of the `VerifyCsrf
 
 In addition to checking for the CSRF token as a POST parameter, the Laravel `VerifyCsrfToken` middleware will also check for the `X-CSRF-TOKEN` request header. You could, for example, store the token in a "meta" tag:
 
-	<meta name="csrf-token" content="{{ csrf_token() }}" />
+	<meta name="csrf-token" content="{{ csrf_token() }}" >
```
Comment (GrahamCampbell) [inline routing.md]: should be no space either?

Comment (antonybudianto) [inline routing.md]: oh, i missed that

File: routing.md [hunk 2]
```diff
@@ -97,18 +97,18 @@ You may constrain the format of your route parameters using the `where` method o
 
 	Route::get('user/{name}', function ($name) {
 		//
-	})
+	});
```
Comment (brunogaspar) [inline routing.md]: This is wrong.

File: routing.md [hunk 3]
```diff
@@ -97,18 +97,18 @@ You may constrain the format of your route parameters using the `where` method o
...
 	->where('name', '[A-Za-z]+');
 
 	Route::get('user/{id}', function ($id) {
 		//
-	})
+	});
```
Comment (brunogaspar) [inline routing.md]: This is wrong.

File: routing.md [hunk 4]
```diff
@@ -97,18 +97,18 @@ You may constrain the format of your route parameters using the `where` method o
...
 	->where('id', '[0-9]+');
 
 	Route::get('user/{id}/{name}', function ($id, $name) {
 		//
-	})
-	->where(['id' => '[0-9]+', 'name' => '[a-z]+'])
+	});
```
Comment (brunogaspar) [inline routing.md]: This is wrong.

**Comment by GrahamCampbell** [comment]
Yeh, @brunogaspar is correct. These semicolons would lead to parse errors.

**Comment by antonybudianto** [comment]
thanks for correction :+1:

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Pull request merged**